openstudiocore/CMake/FindEnergyPlus.cmake

```diff
@@ -50,9 +50,9 @@ else()
   file(GLOB ENERGYPLUS_POSSIBLE_PATHS "/usr/local/[eE]nergy[pP]lus?*")
 endif()
 
-# sort possible paths in reverse order
-list(APPEND ENERGYPLUS_POSSIBLE_PATHS $ENV{ENERGYPLUSDIR})
+# sort possible paths in reverse order, favoring the ENERGYPLUSDIR environment variable
```

Comment:
Do we ever need this finder any more?  Can you use this to create an OpenStudio build with a custom E+ build?